py/src/pathfinder_worker/call.py

```diff
@@ -874,32 +870,25 @@ def create_general_config(chain_id: StarknetChainId) -> StarknetGeneralConfig:
     Separate fn because it's tricky to get a new instance with actual configuration
     """
 
-    weights = resource_fee_weights_0_10_2
-
-    # because of units ... scale these down
-    weights = dict(map(lambda t: (t[0], t[1] * 0.05), weights.items()))
-
-    general_config = StarknetGeneralConfig(
-        starknet_os_config=StarknetOsConfig(chain_id),
-        cairo_resource_fee_weights=weights,
+    return build_general_config(
+        {
+            "cairo_resource_fee_weights": {"n_steps": constants.N_STEPS_FEE_WEIGHT},
+            "contract_storage_commitment_tree_height": constants.CONTRACT_STATES_COMMITMENT_TREE_HEIGHT,
+            "event_commitment_tree_height": constants.EVENT_COMMITMENT_TREE_HEIGHT,
+            "global_state_commitment_tree_height": constants.CONTRACT_ADDRESS_BITS,
+            "invoke_tx_max_n_steps": DEFAULT_MAX_STEPS,
+            "min_gas_price": DEFAULT_GAS_PRICE,
+            "sequencer_address": hex(DEFAULT_SEQUENCER_ADDRESS),
+            "starknet_os_config": {
+                "chain_id": chain_id.name,
+                "fee_token_address": hex(DEFAULT_FEE_TOKEN_ADDRESS),
+            },
+            "tx_version": 1,
```

Comment:
This is the only "variable" constant at this point. I couldn't locate a similar constant in `cairo-lang` itself, and devnet defines its own as well.